File: divisible_and_non_divisible_sums_difference.py
class Solution(object):
    def differenceOfSums(self, n, m):
        """
        :type n: int
        :type m: int
        :rtype: int
        """
        num1=0
        num2=0
        for i in range(1, n+1, 1):
            if(i%m)!=0:num1+=i
            if(i%m)==0:num2+=i
        return num1 - num2

File: divisible_and_non_divisible_sums_difference.java
class Solution {
    public int differenceOfSums(int n, int m) {
        int n1=0;
        int n2=0;
        for (int i=0; i <= n; i++) {
            if ((i%m)!=0){
                n1+=i;}
            if ((i%m)==0){
                n2+=i;}
        }
        return n1 - n2;
    }
}
